spec/fixtures/files/crime_apply_data/applications/148df27d-4710-4c5b-938c-bb132eb040ca.json

```diff
@@ -50,7 +50,7 @@
     "codefendants": [],
     "offences": [
       {
-        "class": "Class C or B",
```

Comment:
Changed this offence to a single class offence as there is another fixture that has identical offence data and changing this meant I could test the overall offence class display for an application with single offence classes that would have an overall offence class. 

Also noticed the offence class key was incorrectly `class` so amended this to `offence_class`

Reply:
Good catch.

Wondering if some (or all) of these fixtures can be replaced with gem fixtures like we do in other places already. I think these fixtures might be just left overs from the old times.
No need to do this in this PR, can be done separately of course or at another time!

Reply:
Ah good to know, I agree they should be replaced so we are consistent in using one approach